p2p/simulations/adapters/docker.go

```diff
@@ -99,7 +98,10 @@ func (d *DockerAdapter) NewNode(config *NodeConfig) (Node, error) {
 	conf.Stack.P2P.NoDiscovery = true
 	conf.Stack.P2P.NAT = nil
 	conf.Stack.NoUSB = true
-	conf.Stack.Logger = log.New("node.id", config.ID.String())
+
+	// listen on a localhost port, which we set when we
```

Comment:
This comment is incorrect, it should say "listen on all interfaces" not localhost.